staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -322,10 +320,25 @@ func (o *DrainCmdOptions) RunDrain() error {
 func (o *DrainCmdOptions) deleteOrEvictPodsSimple(nodeInfo *resource.Info) error {
 	list, errs := o.drainer.GetPodsForDeletion(nodeInfo.Name)
 	if errs != nil {
+		if o.drainer.DryRun {
```

Comment:
I would not print it here, this is error path, so most of the times the list will be empty, and even if it's not running this command w/o dry run would not give you this output so it should not happen. Assume that w/ or w/o dry run you should get more or less the same output.

Reply:
ah, right i was debating that, ok

Reply:
I updated, not sure it's what you're thinking, updated the output example above, also.

Reply:
I've updated